src/gbt_model_builder.pyx

```diff
@@ -18,6 +18,9 @@
 # The model builder object is retrieved through calling model_builder.
 # We will extend this once we know how other model builders will work in DAAL
 
+from daal4py.sklearn._utils import daal_check_version
```

Comment:
I am not sure if this will work in cython src files for daal4py.

It would be better predefine `missing_values_are_supported` in `gbt_model_builder.h`:
```c++
#if __INTEL_DAAL__ >= 2023 && INTEL_DAAL_VERSION >= 20230200
#define missing_values_are_supported 1
#else
#define missing_values_are_supported 0
#endif
```
And import it from the header in this `.pyx` file.

Reply:
I have tried this solution on local machine. It works well. Need I to made more tests?

Reply:
outdated